addons/common/CfgWeapons.hpp

```diff
@@ -0,0 +1,10 @@
+class CfgWeapons {
+    class ItemCore;
+    class InventoryItem_Base_F;
+    class CBA_MiscItem_ItemInfo: InventoryItem_Base_F {
+        type = 302; // "bipod"
+    };
```

Comment:
I'm wondering if this class (`CBA_MiscItem_ItemInfo`) is really needed or if the modified entries couldn't just be put in `CBA_MiscItem/ItemInfo`.

Reply:
It makes configs easier IMHO because you just add the 2 parent defines instead of having to add the full inhertiance from the parent.
Also makes it a drop in replacement for our existing configs (just change `InventoryItem_Base_F`):
```
    class ACE_fieldDressing: ACE_ItemCore {
        class ItemInfo: InventoryItem_Base_F {
            mass = 1;
        };
    };
```